hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/validators/block/BlockContentsValidator.java

```diff
@@ -30,6 +30,18 @@
 import org.apache.logging.log4j.Logger;
 import org.junit.jupiter.api.Assertions;
 
+/**
+ * Validates the structure of blocks. Each block consists of:
+ * 1. A block header
+ * 2. One or more rounds, where each round consists of:
+ *    - A round header
+ *    - One or more event headers or state changes
+ *    - Zero or more transaction groups, where each group consists of:
+ *      a. An event transaction
+ *      b. A transaction result
+ *      c. Zero or more transaction outputs
+ * 3. A block proof
+ */
```

Comment:
This doesn't quite match the current documentation in `block_item.proto`, specifically the state changes, and I'm not sure it's worth trying to specify here. 

```suggestion
 * Validates the structure of blocks matches the expected content and ordering of {@link BlockItem}s
 */
```